hack/rapid-deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -ex
+
+CGO_ENABLED=0 GOOS=linux go build -mod=vendor -ldflags="-X 'main.Version=develop' -buildid=" -trimpath -o linux_controller ./cmd/controller/...
```

Comment:
i called the output `linux_controller` but feel free to tell me that it's not necessary and we should just continue to use the `controller` that was already in the .gitignore.

Reply:
Think it's helpful for users locally to see the difference. I think keeping `linux_controller` makes sense.

Reply:
may be `controller-linux-amd64` a bit better name?